Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2408,6 +2408,8 @@ def qradar_search_create_command(client: Client, params: Dict, args: Dict) -> Co
                                         int(offense_id),
                                         start_time,
                                         return_raw_response=True)
+        if response == QueryStatus.ERROR.value:
```

Comment:
No need to check the response of the `if` block? (and not just the `else` block,) around line 2402.
And then, distinguish the Exception messages.

`if` block Exception message:
```
f'Could not create search for offense_id: {offense_id}'
```

`else` block Exception message:
```
f'Could not create events search for offense_id: {offense_id}'
```